node/core/bitfield-signing/src/lib.rs

```diff
@@ -169,6 +171,12 @@ async fn get_core_availability(
 	Ok(false)
 }
 
+// delegates to the v1 runtime API
+async fn get_availability_cores(_relay_parent: Hash, _sender: &mut mpsc::Sender<FromJob>) -> Result<Vec<CoreState>, Error> {
```

Comment:
I realized this later, but another way to stub this out would be to add the variant to the `RuntimeApiRequest` in the `subsystem` crate.

Reply:
I like that idea; 146746c.